snuba/replacer.py

```diff
@@ -90,10 +89,12 @@ def get_projects_query_flags(project_ids):
 
 
 class ReplacerWorker(AbstractBatchWorker):
-    def __init__(self, clickhouse, dist_table_name, metrics=None):
+    def __init__(self, clickhouse, dataset, metrics=None):
         self.clickhouse = clickhouse
-        self.dist_table_name = dist_table_name
+        self.dataset = dataset
+        self.dist_table_name = dataset.get_schema().get_table_name()
```

Comment:
`dist_table_name` here is misleading as this could be the local table, this should probably be renamed to `table_name`, or possibly change the call site to use `self.dataset.get_schema().get_table_name()` directly.

Reply:
fixed